pkg/network/http/http_statkeeper.go

```diff
@@ -65,6 +65,11 @@ func (h *httpStatKeeper) GetAndResetAllStats() map[Key]RequestStats {
 }
 
 func (h *httpStatKeeper) add(tx httpTX) {
+	if !tx.IsValid() {
+		atomic.AddInt64(&h.telemetry.rejected, 1)
```

Comment:
This changes the meaning of the `rejected` telemetry, which is uses for when a HTTP transaction matches a user-defined filter.

Reply:
If we need a new telemetry, we should add it, rather than confusing what the `rejected` telemetry indicates.